src/utils/TriggerTagsUtils.ts

```
@@ -0,0 +1,36 @@
+    static createTags({
+        domain,
+        retailerId,
+        syncType,
+        reason,
+    }: {
+        domain: string;
+        retailerId?: string;
+        syncType?: 'type_full' | 'type_partial';
+        reason?: string;
+    }): string[] {
+        const tags = [this.createDomainTag(domain)];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)